site/realm/functions/linkReportsToIncidents.js

```diff
@@ -63,8 +63,9 @@ exports = async (input) => {
   await reportsCollection.updateMany(
     {
       report_number: { $in: input.report_numbers },
-      text_inputs: { $in: [null, ""] },
-      text_outputs: { $in: [null, ""] },
+      title: { $nin: [null, ""] },
+      url: { $nin: [null, ""] },
+      source_domain: { $nin: [null, ""] },
```

Comment:
Should this also take into account the new `inputs_outputs` field too?

Reply:
Actually `inputs_outputs` field is not mandatory for a Variant. Actually, Reports can have a `inputs_outputs` field with some values and it is still a Report.